frontend/src/app/shared/components/autocompleter/project-menu-autocomplete/project-menu-autocomplete.template.html

```diff
@@ -47,6 +47,7 @@
 
       <ng-container *ngIf="(loading$ | async) === false; else loadingTemplate">
         <ul
+          *ngIf="(projects$ | async)?.length > 0; else noResultsTemplate"
           op-project-list
           [projects]="projects$ | async"
```

Comment:
can you asign the subscription result to a variable in the ngIf, so you don't have to use async pipe twice?

Reply:
same in the other places